tests: mlp: fix a compile error

alanzhai219 opened the pull request:
# Description

This PR is used to fix a error when compile tests/benchdnn_graph/mlp/mlp.cpp.

# Checklist

## General

- [x] Do all unit and benchdnn tests (`make test` and `make test_benchdnn_*`) pass locally for each commit?
- [x] Have you formatted the code using clang-format?

## Performance improvements

- [N/A] Have you submitted performance data that demonstrates performance improvements?

### New features

- [N/A] Have you published an RFC for the new feature?
- [N/A] Was the RFC approved?
- [N/A] Have you added relevant tests?

### Bug fixes

- [N/A] Have you included information on how to reproduce the issue (either in a github issue or in this PR)?
- [N/A] Have you added relevant regression tests?

## [RFC](https://github.com/oneapi-src/oneDNN/tree/rfcs) PR

- [N/A] Does RFC document follow the [template](https://github.com/oneapi-src/oneDNN/blob/rfcs/rfcs/template.md#onednn-design-document-rfc)?
- [N/A] Have you added a link to the rendered document?

TaoLv commented:
Hi @alanzhai219, thank you for contribution. We already noticed the issue and fixed it internally. I just updated dev-graph branch with the fix. Could you please try if it works for you? Thanks.

alanzhai219 commented:
> Hi @alanzhai219, thank you for contribution. We already noticed the issue and fixed it internally. I just updated dev-graph branch with the fix. Could you please try if it works for you? Thanks.

looks good.not a hard bug. Close it.
